astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 3e044eb26395c8569e3e15bf821c6ae5eeea324a
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
Yes. We updated the astropy helpers upstream. You might have commited after that. Or you need to explictly update. You could do a `git merge upstream/master`